src/identity_committer.rs

```diff
@@ -118,15 +119,12 @@ impl IdentityCommitter {
     async fn process_identities(
         database: &Database,
         identity_manager: &(dyn IdentityManager + Send + Sync),
-        tree_state: &SharedTreeState,
+        tree: &TreeVersion,
```

Comment:
I wanna see this name be more specific about the kind of tree it is: `mined_tree`.